### DISCUSSION TITLE
Adding new record inline demo

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#5755] [Show and tell]
Here i am using Actions to Set a state (addDepartment) and then set conditionals on the add new Container to show / hide.
The container has a nested form - type create.
You can also do a Save + Add combo to add multiple new records before hiding the container again.

This way you can create inline addition of multiple items to a parent.

https://user-images.githubusercontent.com/80512170/167202075-87bb992c-5e5f-4113-9221-4a26aedd9600.mov

**Comment by joebudi** [comment]
This is great. We should really add docs / a tutorial around how to do this. Awesome work and loving the futuristic look!

**Reply by poirazis** [reply]
Thank you , fine sir! 
I'd be more than happy help put these docs together. Just let me know.
